README.md

```diff
@@ -0,0 +1,40 @@
+# Excalidraw
+
+Excalidraw is a whiteboard tool that lets you easily sketch diagrams.
+
+## Try it now
+
+If you want to start sketching, please check https://excalidraw.com
+
+### Excalidraw is built using these awesome tools
+
+- [React](https://reactjs.org/)
+- [Rough.js](https://roughjs.com/)
+
+The project is at its early days, there are some key [features](https://github.com/excalidraw/excalidraw/milestone/1) that we need to implement in order to make the application very useful.
+
+## Getting started
```

Comment:
Can you make the first instruction to go to codesandbox. It's muuuuch easier to contribute as you don't have anything to download/install/configure, you can just change code and see the changes on the right. And even send PRs from there.

- https://codesandbox.io/s/github/excalidraw/excalidraw